risc0/zkvm/methods/build.rs

```diff
@@ -38,7 +38,13 @@ fn main() {
     };
 
     let map = HashMap::from([
-        ("risc0-zkvm-methods-guest", GuestOptions::default()),
+        (
+            "risc0-zkvm-methods-guest",
+            GuestOptions {
+                features: vec![],
+                use_docker: use_docker.clone(),
+            },
+        ),
```

Comment:
Added this as the docker test on the server exec was not using the docker generated elf for the `multi_test`

Reply:
Why are we setting `use_docker` for `risc0-zkvm-methods-std` also?

Reply:
super good catch